src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -659,33 +671,46 @@ const WorkflowView = _.flow(
-                text: `Process multiple workflows from:`,
+                text: `Run workflow(s) with inputs defined by data table`,
                 name: 'process-workflows',
                 checked: this.isMultiple(),
                 onChange: () => this.selectMultiple(),
                 labelStyle: { marginLeft: '0.5rem' }
-              }),
-              h(Select, {
-                'aria-label': 'Entity type selector',
-                isClearable: false, isDisabled: currentSnapRedacted || this.isSingle() || !!Utils.editWorkspaceError(ws), isSearchable: false,
-                placeholder: 'Select data type...',
-                styles: { container: old => ({ ...old, display: 'inline-block', width: 200, marginLeft: '0.5rem' }) },
-                getOptionLabel: ({ value }) => Utils.normalizeLabel(value),
-                value: selectedEntityType,
-                onChange: selection => {
-                  const value = this.updateEntityType(selection)
-                  this.setState({ entitySelectionModel: this.resetSelectionModel(value) })
-                },
-                options: _.keys(entityMetadata)
-              }),
-              h(Link, {
-                disabled: currentSnapRedacted || this.isSingle() || !rootEntityType || !entityMetadata[rootEntityType] || !!Utils.editWorkspaceError(ws),
-                tooltip: Utils.editWorkspaceError(ws),
-                onClick: () => this.setState({ selectingData: true }),
-                style: { marginLeft: '1rem' }
-              }, ['Select Data'])
+              })
             ]),
-            div({ style: { marginLeft: '2rem', height: '1.5rem' } }, [`${this.describeSelectionModel()}`])
+            // !!this.isMultiple() && div({ style: { columns: 'auto', divider: true, ...styles.description, margin: '0.5rem 0 0 2rem' } }, [
+            !!this.isMultiple() && div({ style: { display: 'flex', margin: '0.5rem 0 0 2rem' } }, [
+              div({ style: { marginRight: '2rem' } }, [
+                div({ style: { height: '2rem', fontWeight: 'bold' } }, ['Step 1']),
+                label(['Select root entity type:']),
+                h(Select, {
+                  'aria-label': 'Entity type selector',
+                  isClearable: false,
+                  isDisabled: currentSnapRedacted || this.isSingle() || !!Utils.editWorkspaceError(ws),
+                  isSearchable: true,
+                  placeholder: 'Select data type...',
+                  styles: { container: old => ({ ...old, display: 'inline-block', width: 200, marginLeft: '0.5rem' }) },
+                  value: selectedEntityType,
+                  onChange: selection => {
+                    const value = this.updateEntityType(selection)
+                    this.setState({ entitySelectionModel: this.resetSelectionModel(value, {}) })
+                  },
+                  options: _.map(value => ({ value }), [...entityTypes, ...possibleSetTypes])
```

Comment:
The non-grouped select is ok with a list of strings:
```suggestion
                  options: [...entityTypes, ...possibleSetTypes]
```